core/src/main/kotlin/plugins/OpenApi.kt

```diff
@@ -116,7 +117,9 @@ fun Application.configureOpenApi() {
                     }
                     .connectSubTypes()
                     .handleNameAnnotation()
-                    .generateSwaggerSchema()
+                    .generateSwaggerSchema {
+                        optionalHandling = OptionalHandling.NON_REQUIRED
```

Comment:
Weird that it is not the default to mark optional properties as not required... 😕